pkg/cmd/roachtest/tests/backup.go

```diff
@@ -82,6 +82,11 @@ const (
 	rows3GiB   = rows30GiB / 10
 )
 
+backupTestingBucket := os.Getenv("BACKUP_TESTING_BUCKET")
+if backupTestingBucket == "" {
+        backupTestingBucket="cockroach-bucket-testing-private"
+}
```

Comment:
Is there one central place where this can be checked and set instead of repeating it three times? Then the same variable can be used everywhere.